tt_metal/hw/ckernels/grayskull/metal/llk_api/llk_sfpu/ckernel_sfpu_right_shift.h

```diff
@@ -0,0 +1,26 @@
+// SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include "ckernel.h"
+#include "ckernel_defs.h"
+#include "noc_nonblocking_api.h"
+
+using namespace sfpi;
+
+namespace ckernel {
+namespace sfpu {
+
+//TODO: Implementation pending for GS
+template <bool APPROXIMATION_MODE, int ITERATIONS = 4>
+inline void calculate_right_shift(uint value) {
+#pragma GCC unroll 0
+    for (int d = 0; d < ITERATIONS; d++) {
```

Comment:
Can you please add a static_assert for GS not being supported?
```
static_assert("GS implementation not supported yet");
```

And add the issue number to the assert